### DISCUSSION TITLE
Outreachy Applicant

### DISCUSSION THREAD

**Discussion by vaishnavi-gupta18** [LesFruitsDefendus/saskatoon-ng#198] [Outreachy]
Hello everyone
I am Vaishnavi Gupta, a student pursuing engineering in India. 
I am an Outreachy applicant willing to contribute to open-source. I found Les Fruits défendus collective quite interesting and would love to contribute to it and be part of this community.

**Comment by tristanlatr** [comment]
Hello @vaishnavi-gupta18, thanks for your interest into Saskatoon,

Look forward for your contribution,

Talk to you later,
